.gitignore

```diff
@@ -34,6 +34,9 @@ nosetests.xml
 .cache
 .ropeproject/
 
+# asv environments
+.asv
```

Comment:
Is there any problem if `.asv` is added to `.gitignore`?
A bunch of files in this directory makes my gui-git-client freeze.